plugins/unicodelib/unicodelib.cpp

```diff
@@ -719,6 +720,33 @@ unsigned unicodeEditDistanceV4(UnicodeString & left, UnicodeString & right, unsi
     return da[mask(leftLen-1)][rightLen-1];
 }
 
+void translate(UnicodeString & toProcess, UChar const * sear, unsigned searLen, UChar const * repl, unsigned replLen)
+{
+    UnicodeString search(false, sear, searLen);
+    UnicodeString replace(repl, replLen);
```

Comment:
You could have also change the constructor for this string, and made them both const to emphasises they never changed.